File: Source/Game/Items/Relics/RelicUpgrade.cs
public class RelicUpgrade
{
    public int Level { get; set; }
    public string Description { get; set; }
    public int RequiredLevel { get; set; }
    public int Cost { get; set; }

    public RelicUpgrade(int level, string description, int requiredLevel, int cost)
    {
        Level = level;
        Description = description;
        RequiredLevel = requiredLevel;
        Cost = cost;
    }
}

File: src/Game/Items/Relics/RelicUpgrade.ts
export class RelicUpgrade {
    public Level = 0
    public Description: string
    public RequiredLevel = 0
    public Cost = 0

    public constructor(level: number, description: string, requiredLevel: number, cost: number) {
        this.Level = level
        this.Description = description
        this.RequiredLevel = requiredLevel
        this.Cost = cost
    }
}
